website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx

```diff
@@ -0,0 +1,357 @@
+---
+description: Deploy Appsmith on a Kubernetes cluster
+sidebar_position: 3
```

Comment:
Does this work? We aren't using auto-generated sidebar, right? I don't think this has any affect.

Ref: https://docusaurus.io/docs/sidebar/autogenerated#autogenerated-sidebar-metadata

Auto-generated sidebar is disabled here:
https://github.com/appsmithorg/appsmith-docs/blob/main/website/sidebars.js#L16-L19

Reply:
No, it's simply ignored. Only the description is needed to generate the preview whenever the link is shared.

Reply:
Can we remove it then? Can be misleading to readers right?

Reply:
readers don't see this. It's hidden. But we can remove it. Done.